src/app/components/surveillance/complaints/complaint-edit.jsx

```diff
@@ -32,14 +36,14 @@ import { theme } from 'themes';
 const useStyles = makeStyles(() => ({
   content: {
     display: 'grid',
-    gap: '16px',
+    gap: '32px',
     gridTemplateColumns: '1fr',
     alignItems: 'start',
-    [theme.breakpoints.up('sm')]: {
+    [theme.breakpoints.up('md')]: {
       gridTemplateColumns: '2fr 2fr',
     },
-    [theme.breakpoints.up('md')]: {
-      gridTemplateColumns: '3fr 4fr 4fr 2fr',
+    [theme.breakpoints.up('lg')]: {
+      gridTemplateColumns: '1fr 1fr 1fr .5fr',
```

Comment:
I had always thought that `fr` units that weren't integers were incorrect, so went to figure it out. It turns out I was wrong, but also that this probably shouldn't be a decimal value, according to the w3c: https://www.w3.org/TR/css-grid-1/#fr-unit

Basically, there's a special behavior for fr values between 0 and 1, and unless we want that special behavior, which I think we don't, we shouldn't use fractional fr values. That said, using this change would have the same effect:

```suggestion
      gridTemplateColumns: '2fr 2fr 2fr 1fr',
```

Should also update the other file that has the same `.5fr` value